Microsoft.Toolkit.Uwp.SampleApp/SamplePages/BluetoothLEHelper/BluetoothLEHelperCode.bind

```diff
@@ -0,0 +1,12 @@
+﻿// Get a local copy of the context for easier reading
+BluetoothLEHelper bluetoothLEHelper = BluetoothLEHelper.Context;
+
+// Start the Enumeration
+bluetoothLEHelper.StartEnumeration();
+
+// Connect to a device if your choice
+ObservableBluetoothLEDevice device = 
+bluetoothLEHelper.BluetoothLeDevices[<Device you choose>].Connect()
```

Comment:
Maybe add that as a comment, or even add the code in a "click handler". The way it is right now, it looks one code block and I would expect to simply copy and paste for it work. However, if I run those two lines one after the other, it will probably not work as BluetoothDevices will most likely be empty. Does that make sense?